project-base/storefront/gtm/mappers/getGtmPickupPlaceFromStore.ts

```diff
@@ -19,4 +19,5 @@ export const getGtmPickupPlaceFromStore = (store: StoreOrPacketeryPoint): StoreO
     street: store.street,
     mainImage: null,
     distance: null,
+    specialMessage: null,
```

Comment:
why not store.specialMessage? Same for the distance, why?

Reply:
Its already here :)